chia/rpc/rpc_server.py

```diff
@@ -429,7 +429,8 @@ async def start_rpc_server(
 
         rpc_server = RpcServer.create(rpc_api, rpc_api.service_name, stop_cb, root_path, net_config)
         rpc_server.rpc_api.service._set_state_changed_callback(rpc_server.state_changed)
-        await rpc_server.start(root_path, self_hostname, rpc_port, max_request_body_size)
+        prefer_ipv6 = bool(net_config.get("prefer_ipv6", False))
```

Comment:
`bool()` isn't a great converter.  For example, `bool("false")` is `True` since `"false"` is a non-empty string.  But hey, maybe we look at this as just be explicit about it earlier and someday we'll have a schema that rejects the string `"false"` entirely.

Reply:
Good point, see 5caa80becffb3a53b6d9ecd662c8547944ea925f